Sources/Maintenance/Migration/v3_0/HolidaysToEvents.php

```diff
@@ -679,4 +679,4 @@ public function execute(): bool
 
 		return true;
 	}
-}
+}
```

Comment:
Revert this change

Reply:
I'm not sure what specifically I need to revert here. It looks like I just removed a } and added one back?

This file got messed up when I pulled in the changes from the install/upgrade stuff, Git was marking the file as "changed in both" even though it was new to my branch and I didn't touch it. I ended up copying the raw file from GitHub and replacing my version with that one because I couldn't figure out how to resolve the conflict otherwise.

Reply:
I'm not sure what I can do to fix that but I'll see.